bundlewrap/cmdline/parser.py

```diff
@@ -360,6 +361,39 @@ def build_parser_bw():
         help=_("show config hash for this item on the given node"),
     )
 
+    # bw ipmi
+    help_ipmi = _("Run 'ipmitool' on the ipmit interface of a specific node")
+    parser_ipmi = subparsers.add_parser(
+        "ipmi",
+        description=help_ipmi,
+        help=help_ipmi,
+        formatter_class=RawTextHelpFormatter,  # for HELP_get_target_nodes
+    )
+    parser_ipmi.set_defaults(func=bw_ipmi)
+    parser_ipmi.add_argument(
+        'targets',
+        metavar=_("TARGET"),
+        nargs='+',
+        type=str,
+        help=HELP_get_target_nodes,
+    )
+    parser_ipmi.add_argument(
+        'command',
+        metavar=_("COMMAND"),
+        type=str,
+        help=_("command to run"),
+    )
+    bw_ipmi_p_default = int(environ.get("BW_NODE_WORKERS", "1"))
```

Comment:
Hm, I would have expected `$BW_NODE_WORKERS` to have one default value, not different ones depending on the command. 🤔

Reply:
Yeah, stubled over this as well, will create a follow-up PR to fix this (and some other things that came to mind when reading this file).

Reply:
How are you planning to fix this, what do you have in mind?

Reply:
My plan was to create some more HELP_ variables, and DEFAULT_ variables (like we already have HELP_get_target_nodes) to be used here. (But, as stated, not in this PR)